styles/main.css

```diff
@@ -0,0 +1,764 @@
+    text-align: center;
+    font-weight: bold;
+    color: var(--my-grey);
+    font-size: 0.875rem;
+    margin: 0px;
+}
+
+.home__btns {
+    position: absolute;
+    bottom: 40px;
+}
+
+.home__btn {
+    display: inline-block;
+    padding: 9px 33px;
+    background-color: transparent;
+    color: white;
+    border: 2px solid white;
+    border-radius: 2px;
+    font-size: 0.7rem;
+    font-weight: bold;
+    text-transform: uppercase;
+    text-decoration: none;
+}
+
+.home__learn-more-btn:visited {
+    border: 2px solid rgb(228, 164, 67);
+}
+
+.home__contact-btn:visited {
+    background-color: rgb(228, 164, 67);
+    border-color: rgb(228, 164, 67);
+}
+
+.home__btn:hover,
+.home__btn:active {
+    background-color: white;
+    border-color: white;
+    color: var(--my-green);
+}
+
+.home__contact-btn {
+    display: none;
+}
+
+.services__summary {
+    padding-bottom: 50px;
+    text-align: left;
+}
+
+.services__summary p {
+    margin-bottom: 0px;
+    color: var(--font-main-color);
+}
+
+.services__descriptor {
+    display: grid;
+    grid-template-columns: 18% 82%;
+    grid-template-areas: "img title" ". text";
+    margin-top: 14px;
+}
+
+.services__descriptor--icon {
+    justify-self: center;
+}
+
+.services__descriptor h3 {
+    margin: 0px;
+    align-self: center;
+}
+
+.services__descriptor p {
+    grid-area: text;
+}
+
+.about__video {
+    margin-top: 32px;
+}
+
+#team {
+    text-align: center;
+}
+
+.team__members {
+    padding: 0px;
+}
+
+.team__member {
+    position: relative;
+    margin-bottom: 42px;
+}
+
+.team__member--img {
+    width: 100%;
+}
+
+.team__member--links {
+    background-color: rgb(250, 250, 250);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    position: absolute;
+    z-index: 1;
+    bottom: 16.5%;
+    height: 20%;
+    width: 100%;
+    visibility: hidden;
+    opacity: 0;
+    transition: visibility 0s, opacity 1s ease-out;
+}
+
+.team__member p {
+    font-weight: bold;
+    margin: 0;
+}
+
+.team__member h3 {
+    margin-bottom: 4px;
+}
+
+.portafolio__images {
+    display: flex;
+    flex-direction: column;
+    row-gap: 19px;
+}
+
+.portafolio__image {
+    position: relative;
+}
+
+.portafolio__image img {
+    width: 100%;
+    height: auto;
+}
+
+.portafolio__image--info {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+    position: absolute;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    text-align: center;
+    background-color: rgb(124, 195, 175, 0.8);
+    visibility: hidden;
+    opacity: 0;
+    transition: visibility 0s, opacity 1s ease-out;
+}
+
+.portafolio__image--info h3,
+.portafolio__image--info p {
+    color: white;
+}
+
+.portafolio__image--icon {
+    width: fit-content;
+    font-size: 2rem;
+    color: var(--my-green);
+    background-color: white;
+    border-radius: 50%;
+}
+
+#blog {
+    padding-left: 0px;
+    padding-right: 0px;
+}
+
+#blog>div {
```

Comment:
id in css are used only for specificity like this.
avoid using tags to style, use a class instead